app/webpack/reactjs/helpers/mount.js

```diff
@@ -0,0 +1,12 @@
+import * as ReactDOM from "react-dom";
+import * as React from "react";
+
+export default function mount(Component, mountNodeId) {
```

Comment:
nice

Reply:
I wonder if we're going to need this to be a `global` so it'll be easy to "mount" stuff in the middle of a page that doesn't have a React App - more questions in the final review comment.

Reply:
I'm not sure what "global" here means, if all of our js files are going to be inside of "webpack" folder. Are you thinking of using it in a html file?

Reply:
Yeah - if we need it in the slim file... We already have several javascript classes that act like jQuery plugins.  Those need to be available to `window` because some of them get initialized inside javascript that is inside the slim templates where we don't have access to web pack or `import`.  You can look at an example with `MAU.WhatsThisPopup`.  These get added to `window.MAU` in the `js/globals.js` file.  

I think once I see the example of how we hook this up to a real component on a real page, that'll answer the question about whether or not we need the `mount` method to be surfaced on our `MAU` namespace global.